grub/02_ignition_firstboot

```diff
@@ -10,7 +10,7 @@ search --set=bootpart --label boot
 set ignition_firstboot=""
```

Comment:
I'm pretty sure we can drop this file like on master (#91) since AFAIK 4.3 doesn't use Anaconda on any platform anymore, but I'm just keeping it around to be safe.

Reply:
Yeah...not at all impossible that we may need to rev ignition for 4.2, and while we have separate *downstream* branches for that we don't have separate upstreams.  Dunno.  I guess at this point 4.2 work would be cherry picks downstream anyways?